### PR TITLE
Add new pitot tube and cover

### PR DESCRIPTION
**Pull request by D-ECHO**
related to issue #541 and #492

### CODE REVIEW

File: Models/c172p.xml [hunk 1]
```diff
@@ -6947,9 +6948,10 @@
         <name>rbf-pitot</name>
         <path>Exterior/rbf/rbf.xml</path>
         <offsets>
-            <x-m>-0.220</x-m>
+            <x-m>-0.26</x-m>
             <y-m>-1.525</y-m>
-            <z-m>0.464</z-m>
+            <z-m>0.36wing
```
Comment (onox) [inline Models/c172p.xml]: `0.36wing`?

**Comment by onox** [comment]
@wlbragg Can you check `dev/blender/c172p.blend` and verify `Models/c172p.ac` has been properly exported?

**Comment by onox** [comment]
I think the path to `rbf.png` is absolute instead of relative in `dev/blender/c172p-rbf.blend`.

**Comment by onox** [comment]
@D-ECHO @gilbertohasnofb The new pitot tube looks better than the old one, but I think it should be moved a little bit more to the back. Maybe move it 0.05 in Blender or something?

**Comment by D-ECHO** [comment]
Will deal with that tomorrow afternoon. Good Night

**Comment by onox** [comment]
I noticed the fuselage looks low-res in the new `dev/blender/c172p.blend` file:

d-echo-c172p-blender

Do you have an older version of Blender?

**Comment by wlbragg** [comment]
>@wlbragg Can you check dev/blender/c172p.blend and verify Models/c172p.ac

@onox, I'm not sure what you want me to do? Pull #656 and check the blend and ac?

**Comment by gilbertohasnofb** [comment]
@onox yes, I also think the pitot tube should be moved slightly backwards, see:

**Comment by D-ECHO** [comment]
-Do you have an older version of Blender?,
yes, I have v. 2.69,
-I also think the pitot tube should be moved slightly backwards
ok, will do that
-@D-ECHO Can you make the path relative in dev/blender/c172p-rbf.blend? Then the texture will be loaded correctly on anyone's computer.
I can try
-0.36wing?
Oops, coding mistake ;)

**Comment by onox** [comment]
@D-ECHO Fuselage no longer looks low-res :+1: How did you solve that if I may ask?

About `dev/blender/c172p-rbf.blend`: I didn't see it in the last 2 commits, so path to texture in Blender is still absolute. Did you forget to commit that file? It's the last issue before we can merge this branch.

> @onox, I'm not sure what you want me to do? Pull #656 and check the blend and ac? Actually I don't know how to get this PR, I don't see a branch for it?

@wlbragg See the command line instructions:

```sh
git checkout -b D-ECHO-master master
git pull https://github.com/D-ECHO/c172p-detailed.git master
```

**Comment by D-ECHO** [comment]
@D-ECHO <https://github.com/D-ECHO> Fuselage no longer looks low-res 
:+1: How did you solve that if I may ask?
Actually I dont know ;)
>
> About |dev/blender/c172p-rbf.blend|: I didn't see it in the last 2 
> commits, so path to texture in Blender is still absolute. Did you 
> forget to commit that file? It's the last issue before we can merge 
> this branch.
>
Commited now. Should be relative.

**Comment by wlbragg** [comment]
I've got the c172p.blend open and looking at this picture it looks like we could move the pitot tube back even a little bit farther. It's not easy to tell by the angle of these pictures, is the end of the tube in front, even or behind the leading edge?
o5bcxfb

**Comment by D-ECHO** [comment]
It's a small bit (possibly 1/4) in front of the leading edge

Am 12.01.2016 um 19:13 schrieb wlbragg:

**Comment by wlbragg** [comment]
OK, good to know.
I am comparing the two blends, the one in master and the one in this PR. There are some discrepancies which concern me. For example, on a cursory review, the "point" light is missing from the PR version. Also, a plane.001 (I didn't verify what this is) is also missing in the PR version.

time passes....
I'm looking a little closer now and I see a group that has been created in the original blend that is not grouped in the PR version.

Based on these minor discrepancies I am hesitant to merge this PR without an explanation.

**Comment by onox** [comment]
> Commited now. Should be relative.

The path is relative, but the `//` in front of the path is missing. Via File > User Preferences > File, you can enable "Relative Paths". If you then choose an image in a file selector, Blender should set a relative path beginning with `//`.

> I'm done looking for now. I'm not comfortable merging this. Most of this is minor stuff, but it shouldn't be there at all.

@wlbragg If you're not comfortable merging it, what  we could do is (I) create branch `bug-492`. I'll add @D-ECHO's pitot tube cover model to it. You then copy the new pitot tube (not the cover, but the tube) from @D-ECHO's blender file to some new dummy file, and then add it to the `dev/blender/c172p.blend` in the `bug-492` branch. That way we can still get @D-ECHO's work into master.

**Comment by wlbragg** [comment]
:+1: 

I just completed re-linking all the textures to the c172p.blend's new location. So I will push that at the same time.

**Comment by onox** [comment]
Ok, I'll create `bug-492` in a second.

**Comment by onox** [comment]
@wlbragg Do you want to generate `rfb.ac` or can I just copy one from @D-ECHO?

**Comment by wlbragg** [comment]
I think you can just copy from @D-ECHO

**Comment by onox** [comment]
@wlbragg Pushed `bug-492`.

**Comment by wlbragg** [comment]
OK, done!

**Comment by onox** [comment]
Let me check :smile:

**Comment by wlbragg** [comment]
By the way @D-ECHO, really nice work. It's a big improvement.

**Comment by onox** [comment]
I have updated the position of the cover a little bit so that it fits the pitot tube better. I'll create a new PR.

**Comment by onox** [comment]
I'll close this PR. @D-ECHO Thanks for your work on the cover and tube, much better.

**Comment by onox** [comment]
@D-ECHO You might want to make sure Github knows about your e-mail address that you use in your git commits, then it should be able to create links to your Github account on pages.

**Comment by D-ECHO** [comment]
Thanks ;)

Am 12.01.2016 um 21:51 schrieb wlbragg:

**Comment by wlbragg** [comment]
I don't think I am up for doing this right now, but you know what would be fun for a failure maybe in the future? Simulate a wasp (see above image image) getting stuck in the pitot tube and cause the failure. Have the only way to correct it to use walker to visually inspect the tube and remove the wasp. :laughing:

**Comment by D-ECHO** [comment]
@gilbertohasnofb @wlbragg  can I delete D-ECHO/c172p-detailed now?

**Comment by gilbertohasnofb** [comment]
That's your local fork, so it's up to you what you want to do with that repository (in the sense that it wouldn't affect us or whatsoever).

**Pull request closed without merging**